src/odemis/acq/stream/_sync.py

```diff
@@ -1130,8 +1299,11 @@ def _runAcquisitionEbeam(self, future):
                     raise CancelledError()
                 self._acq_state = FINISHED
 
-            for n, das in enumerate(self._acq_data):
-                self._onCompletedData(n, das)
+            self._im_needs_recompute.clear()
+            # Broadcast all the self._LiveData into self._raw and do post-processing
```

Comment:
I don't think "broadcast" might is the right term. Maybe rephrase as "Adjust the (intermediary) ._live_data to the right shape/format for the final ._raw".